src/modules/site-v2/base/views/tools/genetic_mapping.py

```diff
@@ -115,10 +115,12 @@ def submit():
       # Try submitting the job & returning a JSON status message
       response, code = try_submit(NemascanMapping, user, data, no_cache)
 
-      # If there is an error or a ready message, flash it
-      if not code == 200:
+      # If there was an error, flash it
+      if code != 200 and int(request.args.get('reloadonerr', 1)):

```

Comment:
consider `reloadonerror` instead of shorter version.

Reply:
`200` => HttpStatusCode.OK enum/class?

Reply:
Fixed the first part in [b137d57](https://github.com/AndersenLab/CAENDR/pull/402/commits/b137d5709aff41d74bc8177241ca161ae2e3d02d)